projectDocs/community/expertsList.md

```diff
@@ -0,0 +1,24 @@
+# Community experts list
+
+This document outlines NV Access employees and community contributors.
+This can be used a reference to contact or tag developers in regards to relevant issues and pull requests.
+
+## NV Access
+
+For optimal responses, please avoid tagging NV Access employees on unrelated tickets.
+
+| GitHub | email | topics |
+|---|---|---|
+| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
+| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
+| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |
+| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |
```

Comment:
```suggestion
| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting, Windows accessibility infrastructure and APIs |
```

Reply:
Appreciate the suggestion @codeofdusk - but @SaschaCowley and I are fine to tag on these topics